### ISSUE TITLE
Build failure for Pillow

### ISSUE THREAD

**Issue by bonebox** [SynoCommunity/spksrc#1708]
I get the following error when running `make arch-armadaxp-5.1` in `cross/pillow`:

```
tar -xzpf /home/user1/source/spksrc/cross/pillow/../../distrib/Pillow-2.7.0.tar.gz -C /home/user1/source/spksrc/cross/pillow/work-armadaxp-5.1
===>  Patching for Pillow
patch -p0 < patches/001-fix-paths.patch
patching file setup.py
===>  Configuring for Pillow
===>  Compiling for Pillow
env: setup.py: No such file or directory
../../mk/spksrc.python-module.mk:27: recipe for target 'compile_python_module' failed
make[1]: *** [compile_python_module] Error 127
make[1]: Leaving directory '/home/user1/source/spksrc/cross/pillow'
../../mk/spksrc.cross-cc.mk:126: recipe for target 'arch-armadaxp-5.1' failed
make: [arch-armadaxp-5.1] Error 2 (ignored)
```

**Comment by Dr-Bean** [comment]
Pillow is a Python module, and can only be compiled as part of the Python package.

**Comment by bonebox** [comment]
I compiled the Python package adding Pillow as a dependency, and it still fails with the same error while building Python.

**Comment by Dr-Bean** [comment]
How do you mean, you added Pillow? It's already part of the dependencies: https://github.com/SynoCommunity/spksrc/blob/master/spk/python/Makefile#L10, second entry.
Can you paste the error you're getting when compiling Python?

**Comment by bonebox** [comment]
Yeah, I didn't notice that. Regardless, it still fails with the exact error above.  I found the issue is in `spksrc.python-module.mk` that is included in the Pillow Makefile-- $(HOSTPYTHON) was returning nil.  So I replaced that with the actual path to the native python interpreter and it built successfully.

**Comment by bonebox** [comment]
That said, I realize that's not a long-term solution. The real issue is why is the path not being set for $(HOSTPYTHON).

**Comment by Dr-Bean** [comment]
HOSTPYTHON is defined in `python-cc.mk` as part of compiling `cross/python`. During compilation of `spk/python`, the contents of `python-cc.mk` are used to compile the modules.

I happen to be compiling some arches for Python at the moment, and that's still working as expected here.
So assuming that `cross/python` completed succesfully on your end before continuing with the compilation process, what does your `python-cc.mk` look like? It should be available in `spk/python/work-armadaxp-5.1`.

**Comment by bonebox** [comment]
Apparently my environment was screwed up--I did `make clean` on `cross/python` as well as a few others, and retried `spk/python` and it built successfully this time without having to modify anything.  I was able to successfully compile Pillow with libwebp, which was the point of this exercise.
